scripts/requirements-run-test.txt

```diff
@@ -10,3 +10,6 @@ pyocd>=0.24.0
 
 # used by sanitycheck for board/hardware map
 tabulate
+
+# used by mcuboot
+'cbor>=1.0.0'
```

Comment:
@mbolivar-nordic Is this OK?
cbor package is required by MCUBoot since https://github.com/JuulLabs-OSS/mcuboot/pull/674 was merge

Reply:
typo xD